src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/service/BulkPrintService.java

```diff
@@ -161,6 +164,11 @@ private static Map<String, Object> getAdditionalData(final SscsCaseData sscsCase
         additionalData.put(CASE_IDENTIFIER, sscsCaseData.getCcdCaseId());
         additionalData.put(APPELLANT_NAME, sscsCaseData.getAppeal().getAppellant().getName().getFullNameNoTitle());
         additionalData.put(RECIPIENTS, getRecipients(recipient));
+
+        YesNo isInUk = sscsCaseData.getAppeal().getAppellant().getAddress().getInMainlandUk();
+        if (isInUk != null && !isYes(isInUk)) {
```

Comment:
I don't think you need the null check here

Reply:
The null check was added for the cases that aren't IBA or dont set this field

Reply:
null check is not required as the two methods are fine if you pass null values:
`    public static boolean isYes(YesNo yesNo) {
        return YES.equals(yesNo);
    }

    public static boolean isYes(String yesNo) {
        return nonNull(yesNo) && YES.getValue().equalsIgnoreCase(yesNo);
    }
`

Reply:
Changed the code to be clearer with what I intended it to do when handling non-iba cases